app/src/main/res/layout/patientsummary_bpsummary_content.xml

```diff
@@ -4,28 +4,36 @@
   android:layout_width="match_parent"
   android:layout_height="wrap_content">
 
-  <Button
+  <TextView
     android:id="@+id/addNewBP"
-    style="@style/Widget.AppCompat.Button.Borderless.Colored"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:background="?attr/selectableItemBackground"
     android:drawableStart="@drawable/ic_add_circle_blue1_24dp"
     android:drawablePadding="@dimen/spacing_8"
+    android:paddingStart="@dimen/spacing_8"
+    android:paddingTop="10dp"
```

Comment:
We already have 4 usages of `10dp` in this file. Should we extract it as a separate resource since it is quite likely to be required in the blood sugar view as well?

Reply:
Absolutely. I will do that